ckcp/gitops.sh

```diff
@@ -255,8 +319,8 @@ install_pipelines_crds() {
 install_pipelines_controller() {
   # Create kcp-kubeconfig secret for pipelines controller
   echo -n "  - Register KCP secret to host cluster: "
-  oc create namespace pipelines --dry-run=client -o yaml | oc apply -f - --wait &>/dev/null
-  oc create secret generic kcp-kubeconfig -n pipelines --from-file "$KUBECONFIG_KCP" --dry-run=client -o yaml | oc apply -f - --wait &>/dev/null
+  kubectl create namespace pipelines --dry-run=client -o yaml | kubectl apply -f - --wait &>/dev/null
+  kubectl create secret generic kcp-kubeconfig -n pipelines --from-file "$KUBECONFIG_KCP" --dry-run=client -o yaml | kubectl apply -f - --wait &>/dev/null
```

Comment:
For what it's worth: we will be using Argo CD for the installation of OpenShift Pipelines, which in the case of OpenShift will end up being the creation of a Subscription.

Reply:
According to the discussion yesterday, it will use the kubectl command to create namesapce and `kcp-kubeconfig` secret